solidity/.github/workflows/contracts.yml

```diff
@@ -0,0 +1,122 @@
+name: Solidity
```

Comment:
We need to move the configuration file to `.github/workflows/contracts.yml`.

Reply:
🤦

Reply:
... and remove this file :)